melodies_monet/_cli.py

```diff
@@ -1028,26 +1028,59 @@ def get_aqs(
                     typer.echo("Note that the daily option currently requires monetio >0.2.5")
                 raise
 
-    if not daily:
-        with _timer("Fetching site metadata"):
-            # Need UTC offset in order to compute local time
-            # But currently the `meta=True` option doesn't work
-            meta0 = pd.read_csv(
-                "https://aqs.epa.gov/aqsweb/airdata/aqs_sites.zip",
-                encoding="ISO-8859-1",
-                usecols=[0, 1, 2, 17],
-                dtype=str,
+    with _timer("Fetching site metadata"):
+        # Need UTC offset in order to compute local time
+        # But currently the `meta=True` option doesn't work
+        meta0 = pd.read_csv(
+            "https://aqs.epa.gov/aqsweb/airdata/aqs_sites.zip",
+            encoding="ISO-8859-1",
+            usecols=[0, 1, 2, 17, 24, 25],
+            dtype=str,
+        )
+        meta = (
+            meta0.copy()
+            .assign(
+                siteid=meta0["State Code"] + meta0["County Code"] + meta0["Site Number"],
+                utcoffset=meta0["GMT Offset"].astype(int),
             )
-            meta = (
-                meta0.copy()
-                .assign(
-                    siteid=meta0["State Code"] + meta0["County Code"] + meta0["Site Number"],
-                    utcoffset=meta0["GMT Offset"].astype(int),
-                )
-                .drop(
-                    columns=["State Code", "County Code", "Site Number", "GMT Offset"],
-                )
+            .drop(
+                columns=["Site Number", "GMT Offset"],
+            )
+            .rename(
+                columns={
+                    "State Code": "state_code",
+                    "County Code": "county_code",
+                    "City Name": "city_name",
+                    "CBSA Name": "cbsa_name",
+                }
             )
+        )
+        meta.loc[meta["city_name"] == "Not in a City", "city_name"] = "Not in a city"  # normalize
+
+        counties0 = pd.read_csv(
+            "https://aqs.epa.gov/aqsweb/documents/codetables/states_and_counties.csv",
+            encoding="ISO-8859-1",
+            dtype=str,
+        )
+        counties = (
+            counties0.copy()
+            .rename(
+                columns={
+                    "State Code": "state_code",
+                    "State Name": "state_name",
+                    "State Abbreviation": "state_abbr",
+                    "County Code": "county_code",
+                    "County Name": "county_name",
+                    "EPA Region": "epa_region",  # note without R prefix
+                }
+            )
+        )
+        counties["epa_region"] = "R" + counties["epa_region"]
```

Comment:
AirNow EPA region number isn't zero-padded (R1 instead of R01, etc.). Should be consistent here.

Reply:
Adressed in 5587b6f